### PR TITLE
feat(docs): document SHOW PARTITIONS FROM table

### PR DESCRIPTION
**Pull request by marregui**
documentation for PR https://github.com/questdb/questdb/pull/3060

### CODE REVIEW

File: docs/reference/function/meta.md [hunk 1]
```diff
@@ -143,4 +146,64 @@ wal_tables();
...
+- `partitionBy` - _STRING_, one of _NONE_, _HOUR_, _DAY_, _WEEK_, _MONTH_ and _YEAR_
+- `name` - _STRING_, name of the partition, e.g. `2023-03-14`, `2023-03-14.detached`, `2023-03-14.attachable`
+- `minTimestamp` - _LONG_, min timestamp of the partition (_NaN_ when the table is not partitioned)
+- `maxTimestamp` - _LONG_, max timestamp of the partition (_NaN_ when the table is not partitioned)
+- numRows - _LONG_, number of rows of the partition
+- `diskSize` - _LONG_, size of the partition in bytes
+- `diskSizeHuman` - _STRING_, size of the partition meant for humans to read (same output as function 
+  [size_pretty](/docs/reference/function/numeric/#size_pretty))
+- `readOnly` - _BOOLEAN_, true if the partition was 
+  [attached via soft link](/docs/reference/sql/alter-table-attach-partition/#symbolic-links) (unless it is detached)
```
Comment (amyshwang) [inline docs/reference/function/meta.md:174]: (unless it is detached): I don't understand this. Is it going to be true is a partition is detached too?

File: docs/reference/function/meta.md [hunk 2]
```diff
@@ -143,4 +146,64 @@ wal_tables();
...
+SELECT size_pretty(sum(diskSize)) FROM table_partitions('my_table')
+```
+
+| size_pretty |
+|-------------|
+| 2.8 MB      |
+
+
+```questdb-sql title="Get active partition of a table"
+SELECT * FROM table_partitions('my_table') WHERE active = true
```
Comment (amyshwang) [inline docs/reference/function/meta.md:208]: I think this returns nothing as no partition is active?
Should we indicate the output (NULL?) or text explaining that nothing will be returned? What is the actual behaviour?

File: docs/reference/function/meta.md (hunk 1 above)
Comment (marregui) [reply docs/reference/function/meta.md]: indeed, that is debris, apologies, removed, made no sense.

File: docs/reference/function/meta.md (hunk 2 above)
Comment (marregui) [reply docs/reference/function/meta.md:214]: mistake, I have amended the example to show the correct answer.

File: docs/reference/function/meta.md [hunk 3]
```diff
@@ -168,42 +165,62 @@ Returns a `table` with the following columns:
...
 ```
+
+| index | partitionBy | name       | minTimestamp          | maxTimestamp          | numRows | diskSize | diskSizeHuman | readOnly | active | attached | detached | attachable |
+|-------|-------------|------------|-----------------------|-----------------------| ------- |----------|---------------|----------|--------|----------|----------|------------|
+| 3     | WEEK        | 2023-W03   | 2023-01-16 00:00:00.0 | 2023-01-18 12:00:00.0 | 101     | 83902464 | 80.0 MB       | false    | true   | true     | false    | false      | 
+
+:::info
+
+Use this method when you would like to filter the partitions.
+The quick syntax `SHOW PARTITIONS FROM my_table` does not support filters.
```
Comment (amyshwang) [inline docs/reference/function/meta.md:224]: Add link and move this to an earlier section

Comment (amyshwang) [reply docs/reference/function/meta.md:224]:
```suggestion
The SQL keyword [`SHOW PARTITIONS`](/docs/reference/sql/show/#show-partitions) does not support filters.
```

**Review by amyshwang — CHANGES_REQUESTED** [review]
Are all the queries in `SHOW` not supporting filters? I think we should add a note if this is the case.

File: docs/reference/function/meta.md [hunk 4]
```diff
@@ -148,6 +88,63 @@ wal_tables();
...
+
+| column | type      | indexed | indexBlockCapacity | symbolCached | symbolCapacity | designated |
+|--------|-----------|---------|--------------------|--------------|----------------|------------|
+| symb   | SYMBOL    | true    | 1048576            | false        | 256            | false      |
+| price  | DOUBLE    | false   | 0                  | false        | 0              | false      |
+| ts     | TIMESTAMP | false   | 0                  | false        | 0              | true       |
+| s      | STRING    | false   | 0                  | false        | 0              | false      |
+
+```questdb-sql title="Get designated timestamp column"
+SELECT column, type, designated FROM table_columns('my_table') WHERE designated
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/reference/function/meta.md]: Git blame shows this update - https://github.com/questdb/questdb.io/commit/56878f38111cc1777e0d920f95a824c1d2074003

I guess it should be:

```
SELECT column, type, designated FROM table_columns('my_table') WHERE designated = 'true';
```
Is it correct?

File: docs/reference/function/meta.md [hunk 5]
```diff
@@ -131,7 +131,7 @@ table_columns('my_table')
 | s      | STRING    | false   | 0                  | false        | 0              | false      |
 
 ```questdb-sql title="Get designated timestamp column"
-SELECT column, type, designated FROM table_columns('my_table') WHERE designated
+SELECT column, type, designated FROM table_columns('my_table') WHERE designated = true
```
Comment (amyshwang) [inline docs/reference/function/meta.md:134]: Missing ;

Comment (amyshwang) [reply docs/reference/function/meta.md:134]:
```suggestion
SELECT column, type, designated FROM table_columns('my_table') WHERE designated = true;
```

**Review by amyshwang — APPROVED** [review]

**Pull request merged**